site/layouts/partials/nav.html

```diff
@@ -1,19 +1,25 @@
-<nav class="overflow-x-scroll overflow-x-visible-ns flex justify-between items-center center bg-white divider-grey relative">
+<div class="clip"><a href="#entryTitle">skip to main content</a></div>
+<nav class="nav-collapsible overflow-x-scroll overflow-x-visible-ns flex justify-between items-center center bg-white divider-grey relative" id="js-nav-collapsible">
 
-	<!-- Logo -->
-	<a href="/" class="pa3 db mr4 h-100 w3 flex-none">
-		<img src="/img/logo.svg" alt="Kaldi logo" class="br0 db mb0 w-100"/>
-	</a>
+  <!-- Logo -->
+  <a href="/" class="pa3 db mr4 h-100 w3 flex-none">
+    <img src="/img/logo.svg" alt="Kaldi logo" class="br0 db mb0 w-100"/>
+  </a>
 
-	<!-- Primary Nav -->
-	<ul class="list pl0 flex mv0 b">
-		<li><a href="/products" class="pa3 grey-3 no-underline db">Products</a></li>
+  <!-- Primary Nav -->
+  <ul class="list pl0 flex mv0 b">
+    <li><a href="/products" class="pa3 grey-3 no-underline db">Products</a></li>
 
-		<li><a href="/values" class="pa3 grey-3 no-underline db">Values</a></li>
+    <li><a href="/values" class="pa3 grey-3 no-underline db">Values</a></li>
 
-		<li><a href="/post" class="pa3 grey-3 no-underline db">Blog</a></li>
+    <li><a href="/post" class="pa3 grey-3 no-underline db">Blog</a></li>
 
-		<li><a href="/contact" class="pa3 grey-3 no-underline db">Contact</a></li>
-	</ul>
+    <li><a href="/contact" class="pa3 grey-3 no-underline db">Contact</a></li>
+  </ul>
+
+  <button class="nav-collapsible_toggle" id="navCollapsibleToggle">
```

Comment:
Use Tachyons styling as much as possible, as well as the existing `btn` class. This button should probably look just like the submit button in the footer. It should also be shrunk so that it doesn't make the navigation bar taller (note that "Kaldi" is no longer vertically centered.